BackToBasicsUIKit/BackToBasicsUIKit/Demo ViewControllers/Jimmy/JimmyViewController.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  JimmyViewController.swift
+//  BackToBasicsUIKit
+//
+//  Created by Jimmy Gutierrez on 5/23/19.
+//  Copyright © 2019 SwiftMiami. All rights reserved.
+//
+
+import UIKit
+
+class JimmyViewController: UIViewController, canRecieve {
+
+    @IBOutlet var label: UILabel!
+    @IBOutlet var textField: UITextField!
+    
+    override func viewDidLoad() {
+
+
+    }
+    
+    
+    @IBAction func buttonPressed(_ sender: Any) {
+        performSegue(withIdentifier: "sendDataForwards", sender: self)
```

Comment:
Instead of using a string literal for the identifier here and in the method below, it might be better to declare a private variable to avoid spelling mistakes and the ability to rename this identifier in a single place.

Reply:
Or just don't use segues, they're gross.